src/metaswitch/clearwater/cluster_manager/plugin_utils.py

```diff
@@ -161,12 +161,14 @@ def leave_cassandra_cluster(namespace=None):
 
 
 def start_cassandra():
-    run_command("monit monitor -g cassandra")
+    cassandra_monit_still_to_do = True
 
     # Wait until we can connect on port 9160 - i.e. Cassandra is running.
     s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
     while True:
         try:
+            if cassandra_monit_still_to_do:
+                cassandra_monit_still_to_do = run_command("monit monitor -g cassandra")
```

Comment:
This is a little obscure - I see that it works (because run_command returns the return code, which is 0 if it succeeded and we don't need to retry) but I had to think about it. Could we make this clearer?

```
rc = run_command("monit monitor -g cassandra")
cassandra_monit_still_to_do = (rc != 0)
```